npm/test/getfilecheckprogress.test.ts

```diff
@@ -59,56 +56,71 @@ class GraphqlClientFailure {
   }
 }
 
-test("getFileChecksInfo returns the correct consignment data with a successful api call", (done) => {
+class GraphqlClientErrors {
+  mutation: (
+    query: DocumentNode,
+    variables: GetFileCheckProgressQueryVariables
+  ) => Promise<ExecutionResult> = async (_, __) => {
```

Comment:
Ooh, I've never seen this double underscore in TypeScript before. What does it do?

Reply:
Ah, it's used here the same way as a single underscore i.e. unused/insignificant parameter, it's just that you can not do `(_, _) => foo` like in Scala because the underscores here are just regular parameter names (which have to be unique). I.e. because Javascript allows you to have underscores for variable/parameter names, we just used them to signify these parameters will not be used since we are mocking the response.